### DISCUSSION TITLE
Assasin's Creed

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#41] [Q&A]
Commentaires sur ce quiz

**Accès**
- Serveur de développement (protégé) :  https://timunix3.csfoy.ca/~quiz-TIM/tpred/index.html
- Dépôt Github (privé pour le moment): https://github.com/TPRed/quiz-tim
- Github.io :

**Comment by evefevrier** [comment]
@TPRed 
Beaucoup d'atmosphère dans ce quiz, c'est une belle découverte pour moi qui ne m'intéresse pas beaucoup aux jeux.
Petite observation rapide: au survol des boutons radios tous les choix de réponses bougent. Pour éviter cela on peut prévoir à l'état normal un `border` `transparent` de même épaisseur.

**Comment by ARTADestrokzZ** [comment]
Le style de ton quiz que tu as choisis représente bien l'univers d'Assassin's Creed, avec ces couleurs, ces images et le logo que tu as créé. Le seul point négatif serait les boutons de validation et les boutons pour passer à la prochaine question qui ne sont pas assez gros et peu visible sur le coté gauche. Après c'est juste une préférence.

**Reply by ARTADestrokzZ** [reply]
@TPRed
